### DISCUSSION TITLE
Build is stuck in queued

### DISCUSSION THREAD

**Discussion by karstenwinter** [vercel/community#1432] [Help] [answered]
### Summary

https://vercel.com/pflegevertrauen-team/kundenportal/BHWuvpFk5Ufv33mcY3z1rRn3XDH9

is stuck in queued,
could you have a look why that is the case?

There is no parallel build running.

**Comment by heggemsnes** [comment]
Experiencing this atm as well.

**Comment by karstenwinter** [comment, marked as answer]
solved itself

**Comment by amyegan** [comment]
There was a recent [build processing incident](https://www.vercel-status.com/incidents/3bmz80lvdqzv) that caused builds to get stuck in the queue. The team is working to get queued builds processed and it should be resolved shortly. You can find the latest updates at [vercel-status.com](https://www.vercel-status.com/)
